wordpress-mobile/WordPress-iOS
# Jetpack: update app icon and splash screen

### To test

1. Run the app
2. Check the icon
3. Check the splash screen in light and dark mode

Please also test it on iPad.

### App icon

<img src="https://user-images.githubusercontent.com/7040243/114084159-3bb9bd00-9886-11eb-9ede-001730940955.png" width="300">

### Splash screen

| Light mode | Dark mode |
| ----------- | ----------- |
| ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-08 at 16 12 30](https://user-images.githubusercontent.com/7040243/114083972-0ad98800-9886-11eb-81db-93559ea82b5f.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-08 at 16 12 42](https://user-images.githubusercontent.com/7040243/114084000-1167ff80-9886-11eb-8141-b9f3e968d2ae.png) |

## Regression Notes

These are just image updates on the Jetpack target.

PR submission checklist:
